src/app/components/header/header.component.html

```diff
@@ -1,5 +1,15 @@
 <header class="header">
   <div class="container">
-    <mdb-search class="header--search"></mdb-search>
+    <div class="header--wrap">
```

Comment:
Is there a way to find this page from main page? Probably you could use header everywhere. And also it's good practice to add navigation to main page via logo or something like that

Reply:
Sorry, I found the way. But it's still good to add navigation to main page

Reply:
Added logo to navigate to mainpage.